pkg/apis/options/legacy_options.go

```diff
@@ -2,6 +2,7 @@ package options
 
 import (
 	"fmt"
+	"github.com/oauth2-proxy/oauth2-proxy/v7/pkg/util"
```

Comment:
This should be moved with the rest of the imports for OAuth2 proxy near line 11, linter should pick this up 🤔